src/storage/store/node_column.cpp

```diff
@@ -584,5 +520,7 @@ std::unique_ptr<NodeColumn> NodeColumnFactory::createNodeColumn(const LogicalTyp
     }
 }
 
+NodeColumn::~NodeColumn() = default;
```

Comment:
It had been needed because of the unique_ptr containing a forward declaration of the PhysicalMapping, but is no longer necessary since I removed the PhysicalMapping class.